realm/public.gno

```diff
@@ -95,6 +97,29 @@ func RepostThread(userPostsAddr std.Address, threadid PostID, comment string) Po
 	return repost.id
 }
 
+// Get all users with their address, sorted by name. The response is a JSON string.
+func GetUsers() string {
```

Comment:
What do you think of splitting GetUsers into 2 functions:
1) getUsers() returns an array of user-address
2) GetJsonUsers() formats what getUsers returns into a JSON string

So the client application would call GetJsonUsers().
Doing this splits the responsibility and improves the maintainability of the code I think.

Reply:
I implemented GetUsers and GetJsonUsers.